integration-cli/docker_cli_save_load_unix_test.go

```diff
@@ -41,41 +40,27 @@ func (s *DockerSuite) TestSaveAndLoadRepoStdout(c *check.C) {
 	loadCmd := exec.Command(dockerBinary, "load")
 	loadCmd.Stdin = tmpFile
 
-	if out, _, err = runCommandWithOutput(loadCmd); err != nil {
-		c.Fatalf("failed to load repo: %s, %v", out, err)
-	}
+	out, _, err = runCommandWithOutput(loadCmd)
+	c.Assert(err, check.IsNil, check.Commentf(out))
 
 	after, _ := dockerCmd(c, "inspect", repoName)
 
-	if before != after {
-		c.Fatalf("inspect is not the same after a save / load")
-	}
+	c.Assert(before, check.Equals, after, check.Commentf("inspect is not the same after a save / load"))
```

Comment:
this commentf can be a standard comment on the same line as there are no variables being interpolated.
`// inspect is not the same after a save / load`